### DISCUSSION TITLE
Ensure url() Method is Called in boot() Method of Lockscreen.php to Avoid Null Return Value

### DISCUSSION THREAD

**Discussion by avinmaster** [MarJose123/filament-lockscreen#65] [Ideas]
Some plugins use url of `MenuItem` and it's more often very important. I think, we need to place `->url()` in `boot()` of `Lockscreen.php`

```
pxlrbt\FilamentSpotlight\Actions\RegisterUserMenu::getUrl(): Return value must be of type string, null returned
````

**Comment by melvinmihaylovnulabg** [comment]
I am getting the same issue. I have installed the spotlight plugin https://filamentphp.com/plugins/pxlrbt-spotlight and when i add the lock screen plugin too it gives me this error: Call to undefined method pxlrbt\FilamentSpotlight\Actions\RegisterUserMenu::getUri().
So is it fixed? And if no, how can i fix it?

**Reply by MarJose123** [reply]
Hi, can you submit with the spotlight package  as issue it seems it is not issue on my package.

**Reply by melvinmihaylovnulabg** [reply]
Thanks, i will do that. That's strange that popped up as a problem, when i installed the extension.
